app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,10 @@
+class AdminController < ApplicationController
+  before_action :require_admin
+  layout "admin"
+
+  def require_admin
+    verboten! unless current_user.super_admin?
```

Comment:
I need to find out who started the use of `verboten!`

Reply:
hahaha I did 

I know it's a normal German word but that was very explicitly created with Wolfenstein3D in mind.